glue_jupyter/common/state_widgets/tests/test_viewer_image.py

```diff
@@ -0,0 +1,12 @@
+def test_contour_levels(app, data_image):
+    viewer = app.imshow(data=data_image)
+    widget_state = viewer._layout_layer_options.layers[0]['layer_panel']
+    layer_state = widget_state.layer_state
+    layer_state.level_mode = "Custom"
+
+    widget_state.contour_levels_txt = '1e2, 1e3'
```

Comment:
@astrofrog for some reason this test didn't run, because it's `c_levels_txt` not `contour_levels_txt` (changed that last minute). Do you know why?

Reply:
Do you mean that yo don't understand why the properties start with ``c_``?

Reply:
You can forget this, basically the test didn't test anything before.